hs_app_netCDF/models.py

```diff
@@ -163,9 +163,9 @@ def get_hs_term_dict(self):
         # add new terms for NetCDF res
         hs_term_dict["HS_NETCDF_FILE_NAME"] = ""
         for res_file in self.files.all():
-            fn = res_file.resource_file.name
-            if fn.lower().endswith('.nc'):
-                hs_term_dict["HS_NETCDF_FILE_NAME"] = fn[fn.rfind('/')+1:]
+            f_name, f_ext = get_resource_file_name_and_extension(res_file)
+            if f_ext.lower() == '.nc':
+                hs_term_dict["HS_NETCDF_FILE_NAME"] = f_name + f_ext
```

Comment:
I do not see a use case where one would only use the name part of the file (without the extension).  I would suggest to change the function to return the file name that includes extension (e.g. "ABC.nc").